users/templates/users/profile.html

```diff
@@ -3,6 +3,8 @@
 {% block content %}
 
 <h2>{{ author.user.username }}</h2>
+<h2>{{ author.id }}</h2>
+<h2>{{ user.author.id }}</h2>
```

Comment:
could you remove these in this pull request? they are easy to add back for testing, but I don't want them in the demo